packages/framework/src/hooks/useRegisterBindings.ts

```diff
@@ -70,7 +84,6 @@ export const useRegisterBindings = <T extends { [key: string]: unknown }>(
   }, [
     methods,
     resolvedWidgetId,
-    setWidgetState,
```

Comment:
Why remove this?

Reply:
Not needed and not used in that useEffect